eng/AfterSigning.targets

```diff
@@ -11,7 +11,7 @@
 
   <Target Name="_ZipLanguageServerBinaries" AfterTargets="Pack"  Condition=" '$(OS)' == 'Windows_NT' ">
     <!-- This target is defined in eng/targets/Packaging.targets and included in every project. -->
-    <MSBuild Projects="$(RepoRoot)src\Razor\src\Microsoft.AspNetCore.Razor.Language\Microsoft.AspNetCore.Razor.Language.csproj"
+    <MSBuild Projects="$(RepoRoot)src\Razor\src\Microsoft.AspNetCore.Razor.LanguageServer\Microsoft.AspNetCore.Razor.LanguageServer.csproj"
```

Comment:
This change was made because this target calculates a version number from the M.A.Razor.Language project. However, that project moved, so I changed it to a different project instead.